### PR TITLE
Report exceeded max encoded size in MB if applied

### PR DESCRIPTION
**Pull request by vshlapakov**
Review, please.

### CODE REVIEW

File: scrapinghub/hubstorage/batchuploader.py [hunk 1]
```diff
@@ -226,9 +226,14 @@ def write(self, item):
         data = jsonencode(item)
         if len(data) > self.maxitemsize:
             truncated_data = data[:self.ERRMSG_DATA_TRUNCATION_LEN] + "..."
+            # report maxitemsize in MBs if it's appropriate
+            maxitemsize_msg = '{} bytes'.format(self.maxitemsize)
+            maxitemsize_mbs = self.maxitemsize // 1024**2
+            if maxitemsize_mbs and self.maxitemsize % 1024**2 == 0:
```
Comment (chekunkov) [inline scrapinghub/hubstorage/batchuploader.py]: should be a utility like http://stackoverflow.com/a/1094933

Comment (vshlapakov) [inline scrapinghub/hubstorage/batchuploader.py]: Sure, looks much better, thank you!

File: scrapinghub/hubstorage/utils.py [hunk 2]
```diff
@@ -131,3 +131,21 @@ def apipoll(endpoint, *args, **kwargs):
...
+    Size is rounded to a closest integer value (for simplicity).
+
+    >>> sizeof_fmt(100)
+    '100 B'
+    >>> sizeof_fmt(1024)
+    '1 KiB'
+    >>> sizeof_fmt(1024*1024 + 100)
+    '1 MiB'
+    """
+    for unit in ['', 'Ki', 'Mi']:
```
Comment (chekunkov) [inline scrapinghub/hubstorage/utils.py]: hopefully, KiB/MiB will not confuse some of our less technical users :)

File: scrapinghub/hubstorage/utils.py [hunk 3]
```diff
@@ -131,3 +131,21 @@ def apipoll(endpoint, *args, **kwargs):
...
+        if abs(num) < 1024.0:
+            return "%.0f %s%s" % (num, unit, suffix)
```
Comment (chekunkov) [inline scrapinghub/hubstorage/utils.py]: any reason to have suffix argument? things will become a bit more readable if we remove it

**Pull request merged**